### ISSUE TITLE
Create new endpoint for geometries of cities

### ISSUE THREAD

**Issue by Joxit** [pelias/api#1121]
Hi,

A new endpoint could be interesting.
An endpoint that would send the geometry of a city using Who's On First ID.
Thanks to this we could highlight cities (like [Who's On First spelunker](https://spelunker.whosonfirst.org/id/101751119/) or [Google](https://www.google.fr/maps/place/Paris,+France)).
This could be another microservice (like pip-service or a new one) and should only serve localities/localadmins (because the regions/countries will be too large).
The answer should also be filtered, do not send the raw data from WOF (keep proprieties about geometry and bbox for example).

Example of request with properties: 

```http
GET /v1/place/geom?id=101751119 HTTP/1.1

HTTP/1.1 200 OK
Content-Type: application/json; charset=utf-8

{
  "id": 101751119,
  "type": "Feature",
  "properties": {
    "geom:area":0.012915,
    "geom:area_square_m":105073572.071737,
    "geom:bbox":"2.22407844723,48.8155766678,2.46976095835,48.9021619073",
    "geom:latitude":48.856626,
    "geom:longitude":2.342883,
    "src:geom":"mz",
    "wof:country":"FR",
    "wof:geomhash":"bcec769e4dc183e582c6cd5dbfd4de9a",
    "wof:id":101751119,
    "wof:lang":[
        "fre"
    ],
    "wof:name":"Paris",
    "wof:placetype":"locality",
},
  "bbox": [
    2.224078447233478,
    48.81557666781795,
    2.469760958345428,
    48.90216190732557
],
  "geometry": {"coordinates":[[],[]],"type":"MultiPolygon"}
}
```

**Comment by orangejulius** [comment]
Yes we definitely want this. We've been [talking about it for a long time](https://github.com/pelias/whosonfirst/issues/19). The entire point of the /v1/place endpoint is to return more details, such as geometry, but we never got around to implementing it.

The [venicegeo](https://github.com/venicegeo/pelias-whosonfirst) team has been working on importing geometries into Elasticsearch as part of WOF records. I don't know if they've done a full planet import, in which case they have not had to worry about the 100MB New Zealand polygon. I like your idea of not returning countries, since those geometries do get quite large. We could also create simplified geometries for display in larger cases.

**Comment by Joxit** [comment]
I thought of something else, if data are stored in elasticsearch, you have to remember that the geometries should not be sent when we have search/autocomplete queries like this one: `/v1/search?text=Paris&sources=wof`.
I do not know if elasticsearch can filter the documents before sending a response. If that's not possible it will be the API that will do it, but it will slow down the queries.

**Comment by orangejulius** [comment]
Yes @joxit that's a very good point. The overhead of sending the geometry JSON to the API with each request would be huge.

Fortunately it looks like it's possible to return [only certain files](https://stackoverflow.com/questions/9605292/make-elasticsearch-only-return-certain-fields#9605566). It looks like it's different in Elasticsearch 5 vs 2 though, so we should watch out for that.

**Comment by Joxit** [comment]
That's cool, there is also an exclude source :smile: 

[Here](https://joxit.github.io/pelias-mapbox-gl-js/demo/) is a preview of what it could give using [whosonfirst-data/whosonfirst-data](https://github.com/whosonfirst-data/whosonfirst-data) repository.
